tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "lib": ["es2020"],
+    "target": "esnext",
+    "module": "CommonJS",
```

Comment:
Nothing in this project requires the CJS module system, so you can remove an unnecessary layer of transpilation

Reply:
Tests require CJS. Mocha doesn't like ESM.